File: ApiSuvesaPos/Negocio/Modelo/inventario/tomageneral/tomageneral.vb
Namespace Modelo
    Public Class tomageneral
        Public Property IdToma As Integer
        Public Property Fecha As Date
        Public Property Anulado As Boolean
        Public Property Ajustado As Boolean
        Public Property NumeroAjuste As Integer
        Public Property TomaGeneralDetalle As New List(Of Modelo.tomageneraldetalle)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/tomageneral/tomageneral.cs
namespace Modelo
{
    public class tomageneral
    {
        public int IdToma { get; set; }
        public DateTime Fecha { get; set; }
        public bool Anulado { get; set; }
        public bool Ajustado { get; set; }
        public int NumeroAjuste { get; set; }
        public List<Modelo.tomageneraldetalle> TomaGeneralDetalle { get; set; } = new List<Modelo.tomageneraldetalle>();
    }
}
